From 31772e9ad8575dff60620a5e36d4f5c06bb94a12 Mon Sep 17 00:00:00 2001
From: Bo Yang <bo.yang@amlogic.com>
Date: Tue, 9 Jun 2015 17:49:24 +0800
Subject: [PATCH 5639/5965] PD#107533: backlight: fix dtd mistake

additional commit for the commit below:
commit 78c074c4486dbfec9a25ce890fc693aef372056f
PD#107533: backlight: support multi panel parameter

Change-Id: I265359a438ff9dbc311c7cddbbe403bb788214d3
---
 arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
index a717ca6deda9..6b89d6c31851 100644
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
@@ -793,7 +793,7 @@
 
 		backlight_0 {
 			index = <0>;
-			name = "backlight";
+			name = "backlight_0";
 			bl_en_gpio = "GPIOY_6";
 			bl_en_on = <0>; /** 1=output high, 0=output low **/
 			bl_pwm_port = "PWM_B";
@@ -806,7 +806,7 @@
 			bl_power_off_delay = <200>; /* unit: ms */
 			bl_pwm_freq = <1000>;
 			bl_pwm_duty_max_min = <100 25>; /* 100% - 25%*/
-		}
+		};
 	};
 
 
-- 
2.19.0

